cli/src/main/java/com/github/ruediste1/btrbck/cli/CliMain.java

```diff
@@ -509,15 +523,14 @@ private StreamRepository readAndLockRepository() {
 		if (path == null) {
 			path = Paths.get("").toFile();
 		}
-		StreamRepository repo = streamRepositoryService.readRepository(path
-				.toPath());
+		StreamRepository repo = streamRepositoryService.readRepository(path.toPath());
 		FileChannel f;
 		try {
 			f = FileChannel.open(repo.getRepositoryLockFile(),
 					StandardOpenOption.WRITE);
 			repositoryLock = f.lock(0L, Long.MAX_VALUE, false);
 		} catch (IOException e) {
-			throw new RuntimeException("Error while locking repository", e);
+			throw new DisplayException("Unable to open lock file: "+repo.getRepositoryLockFile());
```

Comment:
Take a look if this suits you:
https://github.com/Polve/btrbck/commit/ce579f1cee931ce1c08ff40314972e176ac32732
There still is the tab/spaces issue though: I usually use two spaces tab.